gravitee-apim-gateway/gravitee-apim-gateway-http/src/main/java/io/gravitee/gateway/http/connector/http/HttpProxyConnection.java

```diff
@@ -181,6 +181,8 @@ protected T handleUpstreamResponse(final AsyncResult<HttpClientResponse> clientR
             clientResponse
                 .headers()
                 .names()
+                .stream()
+                .filter((headerName) -> !headerName.equalsIgnoreCase(HttpHeaders.ACCEPT_ENCODING))
```

Comment:
Not sure it is the right place. Do you want to remove AcceptEncoding header from the request or from the response ?